djsf.json

```diff
@@ -562,6 +562,12 @@
 					"description": "The sequence number of the station on the route.",
 					"id": "#sequenceNumber",
 					"required":true
+				},
+				"externalIdentifier": {
+					"type": "string",
+					"description": "A unique identifier for the station, this could be any external identifier, for example the REC_ORT number of a VDV452 schedule Import.",
+					"id": "#externalIdentifier",
+					"required": false

```

Comment:
Does it make sense to account for different departure and arrival tracks? In this case we would need different fields (similar to the departure/arrival times)

Reply:
i believe that Motion have the information and can easily deliver it to the schedule, but i can not relate if your suggestion is suitable in the current context of ETC, because it can change a the intend plan/impementation in OnBoard.
I believe that @Shyru can decide about that.

Reply:
I think it does not make sense to account for different arrival and departure tracks. There are very very very few rare cases where this happens at all. From the perspective of ETC we need the arrival track, because we need to send the information that we arrived at this track. Also when retrieving connection infos from ETC the arrival track is more important.
So for the sake of simplicity let's don't make this more complicated than needed.